hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/TargetManagementTest.java

```diff
@@ -114,13 +114,14 @@ public void entityQueriesReferringToNotExistingEntitiesThrowsException() {
                 "DistributionSet");
 
         verifyThrownExceptionBy(() -> targetManagement.countByTargetFilterQuery(NOT_EXIST_IDL), "TargetFilterQuery");
-        verifyThrownExceptionBy(() -> targetManagement.countByRsqlAndNonDS(NOT_EXIST_IDL, "name==*"),
+        verifyThrownExceptionBy(() -> targetManagement.countByRsqlAndNonDSAndCompatible(NOT_EXIST_IDL, "name==*"),
```

Comment:
There seem to be missing tests for the _expected_ behavior of following JpaTargetManagement methods:

- `countByRsqlAndNonDSAndCompatible`
- `countByRsqlAndNotInRolloutGroupsAndCompatible`
- `findByTargetFilterQueryAndNotInRolloutGroupsAndCompatible`
- `findByTargetFilterQueryAndNonDSAndCompatible`

Reply:
`findByTargetFilterQueryAndNonDSAndCompatible` seems to be checked in `TargetManagementSearchTest`.

Reply:
that is correct, I did not add tests for those methods since there are (as you already mentioned) tested in other test classes:

- `countByRsqlAndNonDSAndCompatible` -> `AutoAssignCheckerTest.java`
- `countByRsqlAndNotInRolloutGroupsAndCompatible` + `findByTargetFilterQueryAndNotInRolloutGroupsAndCompatible` -> `RolloutManagementTest.java`